test/evergreen.yml

```diff
@@ -272,16 +272,26 @@ functions:
 
             echo "Checking out branch $branch ..."
             git checkout $branch
-            sh reconf
 
             # Java API is removed in newer branches via WT-6675.
             if [ $branch == "mongodb-4.2" ] || [ $branch == "mongodb-4.0" ]; then
+              cd build_posix
+              sh reconf
               ../configure CFLAGS="-DMIGHT_NOT_RUN -Wno-error" --enable-java --enable-python --enable-strict
               (cd lang/python && make ../../../lang/python/wiredtiger_wrap.c)
               (cd lang/java && make ../../../lang/java/wiredtiger_wrap.c)
-            else
+            elif [ $branch == "mongodb-5.0" ] || [ $branch == "mongodb-4.4" ]; then
+              cd build_posix
+              sh reconf
               ../configure CFLAGS="-DMIGHT_NOT_RUN -Wno-error" --enable-python --enable-strict
               (cd lang/python && make ../../../lang/python/wiredtiger_wrap.c)
+            else
+              . test/evergreen/find_cmake.sh
+              if [ -d cmake_build ]; then rm -r cmake_build; fi
+              mkdir -p cmake_build
+              cd cmake_build
+              $CMAKE -DCMAKE_C_FLAGS="-DMIGHT_NOT_RUN -Wno-error" -DENABLE_PYTHON=1 -DENABLE_STRICT=1 -G "${cmake_generator|Ninja}" ../.
+              ninja lang/python/all
```

Comment:
What if `$cmake_generator` is set to make?

Reply:
Good catch, that would be error-prone. Given that invoking the python target is different across `ninja` and `make`, to keep the logic simple, I've re-worked this task to just consistently use `make` in 5068304eb1c364179fedb5dbb3db72805aac0994 (I preferred `make` here as its generally available across our machines).